README.md

```diff
@@ -86,6 +87,15 @@ Nodes are spread across multiple (up to 3) Availability Zones.
 Healthchecks are run by Kubernetes to assess the health of each node. If a node fails 3 times in a row, it is declared unhealthy and a new one put in to take its place.
 Kubernetes with auto heal and prevent any corrupt nodes from replacing existing healthy ones.
 
+## Backups
+
+This is a transient API with no database, no data passing through the API is persisted.
+
+Only the following is persisted:
+- Mutual TLS certificate authority - S3 (versioning enabled)
+- API container Images - ECR (versioning enabled)
+- Application logs - AWS hosted ElasticSearch cluster (retention period of 30 days)
```

Comment:
What about Azure Application Insights?

Reply:
Could become a maintenance burden to list all dependent services. 
There are services we've not touched on like CloudTrail, Prometheus, API Gateway, Sentry .etc
Backups are usually most interesting to the disaster recovery audience, tempted to remove "Application logs" and limit this list to core components?

Reply:
Yeahhhh, happy to go with that!